source/navitiacommon/navitiacommon/utils.py

```diff
@@ -33,7 +33,7 @@
 import logging
 
 street_source_types = ['OSM']
-address_source_types = ['BANO', 'OSM']
```

Comment:
`OSM` cannot be used as an address source any more?
Is this breaking coverages with OSM but no OpenAddress data?

Reply:
we have never been able to load addresses from osm :stuck_out_tongue_winking_eye: 

I think the value has been set a long time ago thinking it would be done in the short term, but since it's not yet the case, I think we can delete the value :smile: